devicetypes/smartthings/zigbee-window-shade-battery.src/zigbee-window-shade-battery.groovy

```diff
@@ -169,56 +153,47 @@ def levelEventHandler(currentLevel) {
 }
 
 def updateFinalState() {
-	def level = device.currentValue("shadeLevel")
+	def level = device.currentValue("level")
 	log.debug "updateFinalState: ${level}"
-
 	if (level > 0 && level < 100) {
 		sendEvent(name: "windowShade", value: "partially open")
 	}
 }
 
 def batteryPercentageEventHandler(batteryLevel) {
+	log.debug "batteryLevel: ${batteryLevel}"
 	if (batteryLevel != null) {
+    	if (isYooksmartOrYookee()) {
```

Comment:
Please indent this if statement by two tabs (use the tab character, not spaces).